index.html

```diff
@@ -0,0 +1,1161 @@
+            </div>
+          </div>
+        </div>
+        <a href="#" class="col-6 d-none d-lg-block display-2 _robotoSlab text-white text-center _lspc-5 text-decoration-none"><div class=" " style="line-height: 0.9em;">Newsweek</div></a>
+        <div class="col-7 col-lg-3 py-2 _fnt18 font-weight-bold _barlowCond text-right">
+          <a href="#" class="  text-white mr-2 text-decoration-none">SIGN IN</a>
+          <a href="#" class="bg-white text-danger pl-3 pr-2 py-1 text-decoration-none">SUBSCRIBE<img src="images/arrow-right-red.png" class="ml-2 mb-1" alt=".."></a>
+          <a href="" class="d-lg-none "><img src="images/menu.png" alt="..." class="mx-3"></a>
+        </div>
+      </div>
+      <!--  -->
+      <div class="header-bottom bg-white position-relative d-none d-xl-block _fnt18">
+        <img src="images/page-fold.png" alt="page-fold" class="page-fold position-absolute" style="right: 0;">
+        <nav class="navbar navbar-expand-xl d-none d-md-block">
+          <div class="collapse navbar-collapse">
+            <ul class="navbar-nav ">
+              <li class="nav-item">
+                <a class="nav-link text-body" href="#">U.S.</a>
+              </li>
+              <li class="nav-item">
+                <a class="nav-link _pipe text-body" href="#">World</a>
+              </li>
+              <li class="nav-item">
+                <a class="nav-link _pipe text-body" href="#"> Business</a>
+              </li>
+              <li class="nav-item">
+                <a class="nav-link _pipe text-body" href="#"> Tech&#x20;&amp;&#x20;Science</a>
+              </li>
+              <li class="nav-item">
+                <a class="nav-link _pipe text-body" href="#"> Culture</a>
+              </li>
+              <li class="nav-item">
+                <a class="nav-link _pipe text-body" href="#"> Newsgeek</a>
+              </li>
+              <li class="nav-item">
+                <a class="nav-link _pipe text-body" href="#"> Sports</a>
+              </li>
+              <li class="nav-item">
+                <a class="nav-link _pipe text-body" href="#"> Health</a>
+              </li>
+              <li class="nav-item">
+                <a class="nav-link _pipe text-body" href="#"> The Debate</a>
+              </li>
+              <li class="nav-item">
+                <a class="nav-link _pipe text-body" href="#"> Vantage</a>
+              </li>
+              <li class="nav-item">
+                <a class="nav-link _pipe text-body" href="#"> Weather</a>
+              </li>
+            </ul>
+          </div>
+        </nav>
+        <div class="_searchForm text-muted _barlow _fnt17 position-absolute">Search</div>
+      </div>
+      
+    </div>
+    
+  </header>
+
+  <main class="container-fluid bg-white bx-0 mx-auto m-0 p-0" style="max-width: 1310px;">
+    <div class="container-fluid m-0 px-3">
+      <div class="row p-0 m-0">
+      <!-- top-left-section -->
```

Comment:
- [ ] Check the top section in the sm breakpoint. It's not displaying the elements as it should (First goes Top Story, then Featured, Opinion, Debate, More Stories, Culture and Travel and Latest News)

- [ ] The width of the top part of the page (the one that includes the mentioned sections) should have almost 100% width in the small breakpoint but on your page it displays differently. Check the width of the images too.

- [ ] On the contrary, in the last sections, you almost have no margins to the sides. Please fix that.

Reply:
Corrected Sections' appearance priority
Corrected elements margins